cmake/interrogate_setup_dot_py.py

```diff
@@ -199,6 +198,47 @@ def setup(*args, **kwargs):
         with open(outfile, 'w') as out:
             out.write('\n'.join(result))
 
+        attrs = dict(
+            name=package_name,
+            version=version,
+            package_dir=package_dir,
+            packages=pkgs,
+            scripts=scripts,
+            py_modules=modules,
+            entry_points=entry_points,
+        )
+
+        from distutils.dist import Distribution
```

Comment:
Since the rest of the code is using `setuptools` I would suggest to use the same package here.